tst/com/amazon/corretto/crypto/provider/test/EvpKeyFactoryTest.java

```diff
@@ -731,7 +731,9 @@ private static Provider getAlternateProvider(String algorithm) {
         // Similarly, JDK doesn't support EdDSA/Ed25519 until JDK15
         || ((algorithm.equals("Ed25519")
                 || algorithm.equals("Ed25519ph")
-                || algorithm.equals("EdDSA"))
+                || algorithm.equals("EdDSA")
+                || algorithm.equals("XDH")
+                || algorithm.equals("X25519"))
             && TestUtil.getJavaVersion() < 15)) {
```

Comment:
these have been around since JDK11, not JDK15. we should break them out into a separate conjunctive clause that checks 

```
      || ((algorithm.equals("XDH")
                || algorithm.equals("X25519"))
            && TestUtil.getJavaVersion() < 11)
```